change/@microsoft-fast-tooling-react-3155902b-9fc3-473e-a935-6733dd742b4c.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "integrate html renderer into creator",
```

Technically this is the change to this package

```suggestion
  "comment": "ensure oneOf and anyOf state in the Form component is not null before updating it after receiving a custom message",
```